src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -76,6 +77,11 @@ private static void setupDrivers(Application application) {
 		compositeDriver.add(driver);
 		DriverFeature.addDriver("Composite Driver", compositeDriver);
 		DriverFeature.updateDriverInfo();
+		MouseClickAdapter mouseClickAdapter = new MouseClickAdapter(application.getFreePanel(), DriverFeature.getDriverManager());
+
+		application.getFreePanel().addMouseListener(mouseClickAdapter);
```

Comment:
it would be clearer if this has been done using:
mouseClickAdapter.enable()
or (the simplest solution) in a constructor.